packages/vscode-extension/package.nls.json

```diff
@@ -171,6 +171,8 @@
     "teamstoolkit.commandsTreeViewProvider.publish.blockTooltip": "Cannot run command when publishing package. Try again when publish is finished.",
     "teamstoolkit.commandsTreeViewProvider.publish.running": "Publishing...",
     "teamstoolkit.commandsTreeViewProvider.publishDescription": "Publish to Teams",
+    "teamstoolkit.commandsTreeViewProvider.publishInDevPortalTitle": "Publish in Developer Portal",
+    "teamstoolkit.commandsTreeViewProvider.publishInDevPortalDescription": "Publish in Developer Portal",
```

Comment:
Is description same with title?

Reply:
> Is description same with title?

I would keep it same for now as this is the same as what we do for "publish to teams" item. Also double confirming with PM. Will update in another PR if we decide to change.